recipes/pvcaptest/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set version = "0.8.0" %}
+
+package:
+  name: pvcaptest
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/captest/captest-{{ version }}.tar.gz
+  sha256: 45d973484208cef9dfb0e35d9e7caeb95257d71736dcf6d9e292481a6dc345c5
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - bokeh >=1
+    - matplotlib >=2
+    - numpy >=1.13.0
+    - pandas >=0.24
+    - pip
+    - setuptools
+    - python >=3.5
+    - python-dateutil >=2.5
+    - scikit-learn >=0.19
+    - statsmodels >=0.8
+    - holoviews >=1.11
+    - pvlib-python >=0.7
```

I think here you just need ``python`` and ``pip`` to package this project
```suggestion
    - pip
    - python >=3.5
```

---

I removed the other packages from host section.